ui_framework/components/_variables.scss

```diff
@@ -63,3 +63,7 @@ $toolBarHeight: 50px;
 $toolBarPadding: 10px;
 $toolBarSectionSpacing: 50px;
 $toolBarItsemSpacing: 10px;
+
+// Form
+$formControlPaddingSides: 12px;
```

Comment:
If you need this level of definition... `horizontalPadding` is probably more specific then `sides`